# A list of semantic objects in ViMantic.
Header header
SemanticObject[] semanticObjects